Scripts/GitUtils.py

```diff
@@ -221,11 +224,11 @@ def _runGitCommand(gitCmdList, workingDir):
                              stdout=subprocess.PIPE)
     output = popen.communicate()[0]
     if popen.returncode != 0: # command error
-      return (False, output)
-    return (True, output)
+      return (False, codecs.decode(output, encoding='utf-8', errors='ignore'))
```

Comment:
In Python 2, I get the error message: "TypeError: decode() takes no keyword arguments"